### PR TITLE
Copyedits to 3.10 What's New

### PR DESCRIPTION
**Pull request by JelleZijlstra**

### CODE REVIEW

**Review by Fidget-Spinner — APPROVED** [review]

File: Doc/whatsnew/3.10.rst [hunk 1]
```diff
@@ -1279,13 +1279,13 @@ Optimizations
...
+* Add micro-optimizations to ``_PyType_Lookup()`` to improve type attribute cache lookup
   performance in the common case of cache hits. This makes the interpreter 1.04 times faster
-  in average (Contributed by Dino Viehland in :issue:`43452`)
+  on average. (Contributed by Dino Viehland in :issue:`43452`)
 
-* Following built-in functions now support the faster :pep:`590` vectorcall calling convention:
+* The following built-in functions now support the faster :pep:`590` vectorcall calling convention:
   :func:`map`, :func:`filter`, :func:`reversed`, :func:`bool` and :func:`float`.
-  (Contributed by Dong-hee Na and Jeroen Demeyerin in :issue:`43575`, :issue:`43287`, :issue:`41922`, :issue:`41873` and :issue:`41870`)
+  (Contributed by Dong-hee Na and Jeroen Demeyer in in :issue:`43575`, :issue:`43287`, :issue:`41922`, :issue:`41873` and :issue:`41870`)
```
Comment (domdfcoding) [inline Doc/whatsnew/3.10.rst]:
```suggestion
  (Contributed by Dong-hee Na and Jeroen Demeyer in :issue:`43575`, :issue:`43287`, :issue:`41922`, :issue:`41873` and :issue:`41870`)
```

Comment (JelleZijlstra) [reply Doc/whatsnew/3.10.rst]: Thanks!

**Comment by ambv** [comment]
@JelleZijlstra, Miss Islington killed herself on this backport, trying to merge all of `main` into 3.10 for some reason. If you can create a manual backport PR for this, I will merge it.

**Comment by ambv** [comment]
OK, I managed to kick the backport PR: it's at https://github.com/python/cpython/pull/25883

**Pull request merged**